packages/core/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -65,34 +65,26 @@ export interface FolderInfo {
 
 // log project details silently
 export async function validateAndLogProjectDetails() {
```

Comment:
Simplified this. There was also an issue where metric not being emitted for non-Java 8/11 projects due to the conditional in the finally block.